people_admin/views.py

```diff
@@ -178,4 +185,38 @@ def new_legislator(request, state):
 @user_passes_test(lambda u: u.has_perm(EDIT_PERM))
 @require_http_methods(["POST"])
 def apply_new_legislator(request):
+    addition = json.load(request)
+    name = addition["name"]
+    delta = DeltaSet.objects.create(
+        name=f"add {name}",
+        created_by=request.user,
+    )
+    NewPerson.objects.create(
+        delta_set=delta,
+        state=addition["state"],
+        district=addition["district"],
+        chamber=addition["chamber"],
+    )
+    return JsonResponse({"status": "success"})
+
+
+@user_passes_test(lambda u: u.has_perm(EDIT_PERM))
+@require_http_methods(["POST"])
+def apply_bulk_edits(request):
+    edits = json.load(request)
+    for person in edits:
+        updates = []
+        delta = DeltaSet.objects.create(
```

Comment:
I think for review purposes it'd be easier if there were one DeltaSet per update, and then add as many PersonDeltas as you need to it.  this currently creates 1 DeltaSet per person, which might overwhelm the GitHub API portion of this where it creates a PR per DeltaSet (and I think it'd be better to review 40 twitter IDs as part of one PR instead of 40 PRs)

Reply:
Makes total sense, made that change!